packages/components/text/src/text.ts

```diff
@@ -0,0 +1,40 @@
+import { buildProps } from '@element-plus/utils'
+import { componentSizes } from '@element-plus/constants'
+import type Text from './text.vue'
+
+import type { ExtractPropTypes } from 'vue'
+
+export const textProps = buildProps({
+  /**
+   * @description text type
+   */
+  type: {
+    type: String,
+    values: ['primary', 'success', 'info', 'warning', 'danger', ''],
+    default: '',
+  },
+  /**
+   * @description text size
+   */
+  size: {
+    type: String,
+    values: componentSizes,
+    default: '',
+  },
+  /**
+   * @description render ellipsis
+   */
+  truncated: {
+    type: Boolean,
+    default: false,
```

Comment:
It seems that the default value can be removed.

Reply:
> It seems that the default value can be removed.

deleted default value.